lib/Controller/FaceController.php

```diff
@@ -81,4 +122,82 @@ public function getThumb ($id, $size) {
 		return $resp;
 	}
 
+	/** @scrutinizer ignore-unused */
```

Comment:
it is not unused anymore?:)

Reply:
I was trying to avoid reports by the if (false). It never worked ..